TestApi/TestApi.IntegrationTests/Features/Users.feature

```diff
@@ -76,4 +76,10 @@ Scenario: Delete User - NoContent
 Scenario: Delete User - NotFound
 	Given I have a valid delete user by user id request for a nonexistent user
 	When I send the request to the endpoint
-	Then the response should have the status NotFound and success status False
\ No newline at end of file
+	Then the response should have the status NotFound and success status False
+
+#Scenario: Create new AD user - Created
+#	Given I have a valid create AD user request
+#	When I send the request to the endpoint
+#	Then the response should have the status Created and success status True
+#	And the details of the new AD user are retrieved
```

Comment:
Ignore this commented out test, I plan to remove it soon along with lots of other integration tests and move them to unit tests instead.